webpack.common.config.js

```diff
@@ -94,9 +94,6 @@ module.exports = {
         from: './assets/fallbackConfig.js',
         to: 'fallbackConfig.js'
       }],
-    }),
-    new SimpleProgressWebpackPlugin({
```

Comment:
Please remove the import too if not used anymore. Also consider to remove it from the package.json if its not used in other environements.

Reply:
Dependency removed in 4e4fe60a35013b904566cf6cde1a58945a7479eb.